### PR TITLE
Ensure that the avatar clip element doesn't affect page layout

### PR DESCRIPTION
**Pull request by robintown**
For https://github.com/vector-im/element-web/issues/26209

### CODE REVIEW

**Review by weeman1337 — APPROVED** [review]
Tested on the deployment. Works :+1: 

I cannot reproduce the linked issue :shrug: But if this helps, go for it!

File: src/components/Avatar/Avatar.module.css [hunk 1]
```diff
@@ -131,3 +131,9 @@ button.avatar:disabled {
 .stacked-avatars .avatar:not(:first-child) {
   margin-left: calc(var(--cpd-avatar-size) * -0.2);
 }
+
+.clip-path {
+  /* The SVG is invisible, but we still need to ensure it doesn't affect the
+  page's layout */
+  position: fixed;
```
Comment (germain-gg) [inline src/components/Avatar/Avatar.module.css:138]: `left: -9999px` too for good measure? Feel like it might not always be transparent?

**Review by germain-gg — APPROVED** [review]

File: src/components/Avatar/Avatar.module.css (hunk 1 above)
Comment (robintown) [reply src/components/Avatar/Avatar.module.css:138]: Sure, that wouldn't hurt

**Comment by robintown** [comment]
Percy failure is due to the org hitting plan limits, bypassing

**Pull request merged**